### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by eightfiftytwo**
Hi! This is my first time contributing to an FOSS project. I updated the translations for Traditional and Simplified Chinese. Please let me know if I messed something up. Thank you.

### CODE REVIEW

**Comment by manuel-192** [comment]
Thanks for the contribution!

I'm just wondering about the files you have changed. In the commits there are changes to more than one package. And for example, the WELCOME_FONT variable usage looked missing quotes.
Also, when I cloned your repository, I see changes only in package eos-translations.

So I'm a bit confused...

**Comment by eightfiftytwo** [comment]
Originally, I changed both the translation and the Welcome app, since the Chinese text look squished and misaligned on my machine. I thought there was a problem with the default Chinese font. After fiddling around with configs I found that was not the case,  so I reverted the commits to the Welcome app. 
TLDR: I only intend to change the translations.

**Comment by manuel-192** [comment]
OK, thanks for the clarification.
I'll merge the translations soon.
